orange/src/main/java/com/hartwig/hmftools/orange/report/chapters/QualityControlChapter.java

```diff
@@ -256,4 +257,36 @@ private void addSageBQRPlots(@NotNull Document document)
         tumorImage.setHorizontalAlignment(HorizontalAlignment.CENTER);
         document.add(tumorImage);
     }
+
+    private void addTumorStats(@NotNull Document document)
+    {
+        Cells cells = new Cells(reportResources);
+        Table tumorStats = Tables.createContent(contentWidth(),
+                new float[] { 3, 1 },
+                new Cell[] {
+                        cells.createHeader(Strings.EMPTY),
+                        cells.createHeader(Strings.EMPTY),
+                });
+
+        tumorStats.addCell(cells.createContent("Tumor maximum diploid proportion"));
+        tumorStats.addCell(cells.createContent(formatPercentage(report.purple().tumorStats().maxDiploidProportion())));
+
+        tumorStats.addCell(cells.createContent("Number of hotspot mutations"));
+        tumorStats.addCell(cells.createContent(String.valueOf(report.purple().tumorStats().hotspotMutationCount())));
+
+        tumorStats.addCell(cells.createContent("Number of hotspot structural variants"));
+        tumorStats.addCell(cells.createContent(String.valueOf(report.purple().tumorStats().hotspotStructuralVariantCount())));
+
+        tumorStats.addCell(cells.createContent("Sum of small variants allele read counts"));
+        tumorStats.addCell(cells.createContent(String.valueOf(report.purple().tumorStats().smallVariantAlleleReadCount())));
+
+        tumorStats.addCell(cells.createContent("Sum of structural variant tumor fragment counts (excluding single breakends)"));
+        tumorStats.addCell(cells.createContent(String.valueOf(report.purple().tumorStats().structuralVariantTumorFragmentCount())));
+
+        tumorStats.addCell(cells.createContent("Sum of B-allele frequency points in germline diploid regions regions with tumor ratio < 0.8 OR > 1.2"));
+        tumorStats.addCell(cells.createContent(String.valueOf(report.purple().tumorStats().bafCount())));
+
+        document.add(new Tables(reportResources).createWrapping(tumorStats,
+                "Detection of tumor"));
```

Comment:
If we are going to add a subtitle “At least one requirement needs to be met .. “, I think we should add the requirements as well to the table. Maybe you could add these between parentheses like was written in the jira ticket: “Number of hotspot mutations (should be > 0)”.

Reply:
Yes we did have something like that in an earlier version, I took it out because I had to hardcode the thresholds to be consistent with Purple. But agree it would easier for someone to interpret with those thresholds displayed. Guess I can add them back.